### ISSUE TITLE
did:ether driver error:500 Internal Server Error

### ISSUE THREAD

**Issue by caict-develop-zhangbo** [decentralized-identity/universal-resolver#414]
've recently deployed the DID Universal Resolver locally and encountered an error when trying to resolve did:ether. The error message received is as follows:

{"@context":"https://w3id.org/did-resolution/v1","didDocument":null,"didResolutionMetadata":{"error":"internalError","errorMessage":"Driver cannot retrieve result for did:ethr:0x03fdd57adec3d438ea237fe46b33ee1e016eda6b585c3e27ea66686c2ea5358479: 500 Internal Server Error ()","contentType":"application/did+ld+json"},"didDocumentMetadata":{}}

I'm reaching out to inquire if anyone has insights into the possible reasons behind this error. Any assistance or guidance would be greatly appreciated.

**Comment by peacekeeper** [comment]
Hello @caict-develop-zhangbo  thanks for reporting this, but when I try it, it seems to work fine:
https://dev.uniresolver.io/#did:ethr:0x03fdd57adec3d438ea237fe46b33ee1e016eda6b585c3e27ea66686c2ea5358479

Could you try again and let us know if it still fails?

**Comment by caict-develop-zhangbo** [comment]
Yes, the problem still exists. The scenario to reproduce it is when I deploy a  universal-resolver locally and resolve did:ethr:0x03fdd57adec3d438ea237fe46b33ee1e016eda6b585c3e27ea66686c2ea5358479. It works fine when I directly call https://dev.uniresolver.io/#did:ethr:0x03fdd57adec3d438ea237fe46b33ee1e016eda6b585c3e27ea66686c2ea5358479, but it doesn't work with my local deployment, and I don't know where the problem lies."

**Comment by peacekeeper** [comment]
I just tried it, and I can't reproduce this, for me it's working fine.

The Universal Resolver uses this driver for `did:ethr`:
https://github.com/uport-project/uport-did-driver

See configuration here:
- https://github.com/decentralized-identity/universal-resolver/blob/main/docker-compose.yml#L99
- https://github.com/decentralized-identity/universal-resolver/blob/main/uni-resolver-web/src/main/resources/application.yml#L66

Can you maybe check if the `uport/uni-resolver-driver-did-uport:5.0.0` container is running locally, and what logs it shows? And any logs from the `universalresolver/uni-resolver-web:latest` container?

**Comment by mirceanis** [comment]
@peacekeeper This looks like an outdated issue. Should probably be closed.
